### ISSUE TITLE
Add Hindi website translation

### ISSUE THREAD

**Issue by ro31337** [ro31337/libretaxi-www#35]
Bot was translated to Hindi language here: https://github.com/ro31337/libretaxi/issues/537 and we need to translate the website (http://libretaxi.org/) into Hindi the same way it was done for other languages: by translating `index.html` in this repository.

For translator: feel free to create PR right to master, feel free to skip updating links on other pages if you don't have time (I will update them). You just need to translate and commit 1 page only.

**Comment by srishag** [comment]
Is this project active? I can help!

**Comment by ro31337** [comment]
Yes @srishag , it's active and we're looking forward for your help with Hindi translation!

**Comment by srishag** [comment]
[index-hi.txt](https://github.com/ro31337/libretaxi-www/files/1267896/index-hi.txt)

**Comment by ro31337** [comment]
Thanks, that looks great, I'll update translation later today or tomorrow!
